content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Announcing LocalStack 3.0 General Availability!
+description: We are excited to announce the release of LocalStack 3.0 featuring enhanced AWS services, Chaos Engineering tools, IAM Policy Stream, Desktop Application, improved performance and more!
+lead: We are excited to announce the release of LocalStack 3.0 featuring enhanced AWS services, Chaos Engineering tools, IAM Policy Stream, Desktop Application, improved performance and more!
+date: 2023-11-13
+lastmod: 2023-11-13
+images: ['localstack-3.0.png']
+contributors: ["LocalStack Team"]
+tags: ['news']
+show_cta_1: true
+leadimage: 'localstack-3.0.png'
+---
+
+{{< img-simple src="localstack-3.0.png" >}}
+
+Today, we are excited to announce **LocalStack 3.0**, our latest major release towards empowering developers with a high-fidelity local cloud development & testing experience. Earlier this year, we unveiled [LocalStack 2.0](https://localstack.cloud/blog/2023-03-29-announcing-localstack-2.0-general-availability/), which brought forth a host of new features and improvements, along with better integration with the AWS tooling ecosystem. With the 3.0 release, we're doubling down on our promise to boosting developer productivity and tackling the issues that often slow down cloud software development and delivery workflows — specifically, simplifying onboarding and eliminating the inefficiencies that lead to prolonged dev&test cycles. Additionally, this release represents a significant milestone in our journey as we move beyond traditional cloud emulation, and introduce innovative features that promise to reshape the way developers construct resilient cloud applications.
+
+Whether you’re a LocalStack user or just getting started on your cloud development journey, we have a lot of exciting features to share with you! With LocalStack, you can now:
+
+- Test disaster recovery simulations such as region failover, DNS failover, and service failures.
+- Explore and progressively tighten security around IAM policies as your application takes shape.
+- Create ephemeral cloud sandboxes for testing your infrastructure locally or on CI pipelines.
+- Activate analytics for LocalStack in CI runs to get insights into your infrastructure test runs.
```

Comment:
ci projects is not released to the public yet, but the point is vague enough so that we can just leave it in here.